bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/file_processor_rvt.cpp

```diff
@@ -303,6 +279,73 @@ void setupViewOverrides(OdBmDBViewPtr pView)
 	ODBM_TRANSACTION_END()
 }
 
+OdBmDBView3dPtr createDefault3DView(OdBmDatabasePtr pDb, std::string style)
+{
+	// (This implementation is based on the BmDBView3dCreateCmd.cpp TB_Commands
+	// example in the BimRv samples.)
+
+	OdBmDBView3dPtr pDBView3d; // (ODBM_TRANSACTION_BEGIN/END actually creates a try..catch block, so this must be defined outside for it to be returned.)
+
+	ODBM_TRANSACTION_BEGIN(viewCreate, pDb)
+	viewCreate.start();
+
+	pDBView3d = OdBmDBView3d::createObject();
+	pDb->addElement(pDBView3d); // element must be added to DB before setDefaultOrigin, createDefaultDrawingAndViewport, setPerspective
+
+	if (style == "wireframe") {
+		pDBView3d->setDisplayStyle(OdBm::ViewDisplayStyle::Wireframe);
+	}
+	else if (style == "hiddenline") {
+		pDBView3d->setDisplayStyle(OdBm::ViewDisplayStyle::HiddenLine);
+	}
+	else if (style == "shaded") {
+		pDBView3d->setDisplayStyle(OdBm::ViewDisplayStyle::Shaded);
+	}
+	else {
+		pDBView3d->setDisplayStyle(OdBm::ViewDisplayStyle::ShadedWithEdges);
+	}
+
+	pDBView3d->setDetailLevel(OdBm::ViewDetailLevel::Fine);
+	pDBView3d->setViewDiscipline(OdBm::ViewDiscipline::Architectural); // Architectural shows all elements (search "About the View Discipline" in the Revit docs)
```

Comment:
More of a sanity check/question than request for change: I guess we're still hiding annotations as per #767 because we filter them out further down the line?